master/getting-started/kubernetes/installation/calico.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Installing Calico on Kubernetes with Calico networking
+canonical_url: 'https://docs.projectcalico.org/v3.1/getting-started/kubernetes/installation/calico'
+---
+
+## Before you begin
+
+- Ensure that you have a Kubernetes cluster that meets the 
+  {{site.prodname}} [system requirements](../../requirements). If you don't, 
+  follow the steps in [Using kubeadm to create a cluster](http://kubernetes.io/docs/getting-started-guides/kubeadm/) to create one.
+
+- If your cluster has RBAC enabled, issue the following command to 
+  configure the roles and bindings that {{site.prodname}} requires.
+
+   ```
+   kubectl apply -f \
+   {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/rbac-kdd.yaml
+   ```
+   > **Note**: You can also 
+   > [view the manifest in your browser](hosted/rbac-kdd.yaml){:target="_blank"}.
+   {: .alert .alert-info}
+   
+## Installing {{site.prodname}} with {{site.prodname}} networking
+
+### About installing {{site.prodname}} with {{site.prodname}} networking
+
+The procedure differs according to the type of datastore you want {{site.prodname}}
+to use and the number of nodes. Refer to the section that matches your configuration.
+
+- [etcd datastore](#etcd-datastore)
+
+- [Kubernetes API datastore with 50 nodes or less](#kubernetes-api-datastore-with-50-nodes-or-less) (beta)
+
+- [Kubernetes API datastore with more than 50 nodes](#kubernetes-api-datastore-with-more-than-50-nodes) (beta)
+
+> **Note**: {{site.prodname}} networking with the Kubernetes API datastore is 
+> in beta due to the following.
+> - It does not yet support {{site.prodname}} IPAM. We recommend using 
+>   `host-local` IPAM in conjunction with Kubernetes pod CIDR assignments,
+>   as specified in our manifests.
+> - You must use `calicoctl` to configure the node-to-node mesh, default 
```

Comment:
This second bullet is true for Calico in general, and isn't a limitation :)

Let's just drop this one.